engine/frontend/src/component/CreateEnclaveView.js

```diff
@@ -27,38 +29,47 @@ export const CreateEnclaveView = ({packageId, enclave, args}) => {
         }
     }
 
-    useEffect(() => {
-        setLoading(true)
-        let stream;
-        const fetchLogs = async () => {
-          stream = await runStarlark(enclave.host, enclave.port, packageId, args, appData.jwtToken, appData.apiHost);
-          for await (const res of stream) {
-              const result = res["runResponseLine"]
-              if (result.case === "instruction") {
-                  setLogs(logs => [...logs, result.value.executableInstruction])
-              }
+    const readStreamData = (result) => {
+        if (result.case === "instruction") {
+            setLogs(logs => [...logs, result.value.executableInstruction])
+        }
 
-              if (result.case === "progressInfo" && result.value.currentStepInfo.length > 0) {
-                  console.log("progressinfo: ", result)
-                  setLogs(logs => [...logs, result.value.currentStepInfo[result.value.currentStepNumber]])
-              }
+        if (result.case === "progressInfo" && result.value.currentStepInfo.length > 0) {
```

Comment:
@Peeeekay can we use constants for these cases?

Reply:
not using this file; I am using log/CreateEnclaveLog.js and it has this change in it.

Reply:
@Peeeekay if we are not using this file shouldn't we delete it?

Reply:
yeah deleted the file